bin/embedding-sdk/fixup-types-after-compilation.js

```diff
@@ -14,14 +14,12 @@ const SDK_PACKAGE_NAME = "@metabase/embedding-sdk-react";
 
 // this map should be synced with "tsconfig.sdk.json"
 const REPLACES_MAP = {
-  "metabase-enterprise/": `${SDK_PACKAGE_NAME}/dist/enterprise/frontend/src/metabase-enterprise/`,
-  "metabase-lib/": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase-lib/`,
+  "metabase-enterprise": `${SDK_PACKAGE_NAME}/dist/enterprise/frontend/src/metabase-enterprise`,
   "metabase-lib": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase-lib`,
-  "metabase-shared/": `${SDK_PACKAGE_NAME}/dist/frontend/src/metabase-shared/`,
```

Comment:
We no longer references `metabase-shared`

Reply:
Are we sure there may not be a change in the main app that could cause an import from there?

Reply:
@npretto I think it is actually not referenced by any JS/TS file. From full-text search, the only reference is inside a comment (out-of-date), a webpack file, and from pulse:

```
(def ^:private js-file-path "frontend_shared/color_selector.js")
```

![CleanShot 2567-09-05 at 16 24 29@2x](https://github.com/user-attachments/assets/eb0d0db3-4970-436b-bd43-3ba44ee48991)